views/includes/check-your-answers/beneficial-owner-individual.html

```diff
@@ -99,7 +99,11 @@
 {% endset %}
 
 {% if pageParams.isRegistration %}
-  {% set changeLinkUrl = OE_CONFIGS.BENEFICIAL_OWNER_INDIVIDUAL_URL + "/" + boi.id %}
+  {% if not IS_REDIS_REMOVAL_ENABLED %}
```

Comment:
Again, `not` should be removed and logic reversed.

Reply:
To paraphrase a previous comment: The `not` ensures that the larger chunk of the `if` statement is at the bottom (in the `else` clause) and not the top, which makes for cleaner, better formatted code (and avoids having top-heavy conditionals).

Reply:
To paraphrase a previous comment: The `not` ensures that the larger chunk of the `if` statement is at the bottom (in the `else` clause) and not the top, which makes for cleaner, better formatted code (and avoids having top-heavy conditionals).